qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -353,7 +353,10 @@
 <!-- #samples and #columns -->
 <div class="row">
   <div class="col-md-12">
-    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.
+    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.</br>
+    {% if ebi_experiment_accessions %}
+      <b>EBI status: </b> Submitted
```

Comment:
We have other statuses available right? Any reason why we are not showing the current status instead of just Submitted?